doc/components.md

```diff
@@ -208,3 +208,41 @@ The corners are drawn at the same size, the sides are stretched on the side dire
 Using this, you can get a box/rectangle that expands well to any sizes. This is useful for making panels, dialogs, borders.
 
 Check the example app `nine_tile_box` details on how to use it.
+
+## Effects
+
+Flame provides a set of effects that can be applied to certain type of components, these effects can be used to animate some properties of your components, check bellow a list of those effects.
```

Comment:
to certain type -> "to certain types" or "to a certain type"
bellow a list -> below for a list